Mathlib/Data/Nat/Bitwise.lean

```diff
@@ -59,7 +59,6 @@ lemma bitwise_zero_right (n : Nat) : bitwise f n 0 = if f true false then n else
   simp only [ite_self, decide_False, Nat.zero_div, ite_true, ite_eq_right_iff]
   rintro ⟨⟩
   split_ifs <;> rfl
-#align nat.bitwise_zero_right Nat.bitwise_zero_right
```

Comment:
This lemma actually exists in mathlib3, so we should keep the align.

Reply:
Ah, I went by the mismatched diff you posted before. I guess the lemma got removed in the port, but it did exist before?

Reply:
The only mismatch is now `nat.bitwise_bit_aux`. This is an auxiliary theorem which the new `Nat.bitwise_bit` no longer needs, so I removed it. Does the "mathlib should not lose theorems" rule also apply to such purely auxiliary results?

Reply:
I have no idea what happened!

Reply:
Removing auxiliary theorems is ok, but you should use `#noalign`

Reply:
Allright, I've added the noalign, it should match up now.